assignment-02/02-1.py

```diff
@@ -54,24 +59,31 @@ def random_world():
 
 def step(current_state):
     next_state = blank_world()
```

Comment:
next_state is no longer needed then?

Reply:
yeah, it's just one cell at a time, so the results are return in a list by the pool map